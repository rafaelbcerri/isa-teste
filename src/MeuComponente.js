function MeuComponente({ isEnabled }) {
  return (
    <div>
      Banana
      {
        isEnabled && (
          <div>
            SAVE
            {
              [].map((item) => <div>{item.name}</div>)
            }
          </div>
        )
      }
    </div>
  )
}

export default MeuComponente;