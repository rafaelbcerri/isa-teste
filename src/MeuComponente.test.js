import { render, screen } from '@testing-library/react';
import MeuComponente from './MeuComponente';



test('when there is no data', () => {
  const { container } = render(<MeuComponente />);
  expect(container).toMatchSnapshot();
});

test('when there is data', () => {
  const { container } = render(<MeuComponente isEnabled />);

  // expect(screen.getByText('PEixeinho')).toBeInTheDocument();
  // expect(screen.getByTestId('PEixeinho')).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
