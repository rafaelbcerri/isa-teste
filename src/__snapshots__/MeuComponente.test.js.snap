// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when there is data 1`] = `
<div>
  <div>
    Banana
    <div>
      SAVE
    </div>
  </div>
</div>
`;

exports[`when there is no data 1`] = `
<div>
  <div>
    Banana
  </div>
</div>
`;
